lib/fpm/command.rb

```diff
@@ -233,6 +233,17 @@ def help(*args)
     "copying, downloading, etc. Roughly any scratch space fpm needs to build " \
     "your package.", :default => Dir.tmpdir
 
+  option "--source-date-epoch", "SOURCE_DATE_EPOCH",
+    "Integer unix timestamp to use for generated files. " \
+    "See https://reproducible-builds.org/specs/source-date-epoch",
+    :environment_variable => "SOURCE_DATE_EPOCH"
+
+  option "--build-path-prefix-map", "BUILD_PATH_PREFIX_MAP",
+    "If SOURCE_DATE_EPOCH is given without BUILD_PATH_PREFIX_MAP, " \
```

Comment:
Ahh, makes sense. Is this purely about compilers? Which compiler(s)? Would it be useful to instruct users on which software this affected? (gcc, for example, but not javac perhaps?)